digdag-standards/src/test/java/io/digdag/standards/command/ecs/DefaultEcsClientTest.java

```diff
@@ -0,0 +1,54 @@
+package io.digdag.standards.command.ecs;
+
+import com.amazonaws.services.ecs.AmazonECSClient;
+import com.amazonaws.services.ecs.model.ListTagsForResourceRequest;
+import com.amazonaws.services.ecs.model.ListTagsForResourceResult;
+import com.amazonaws.services.ecs.model.Tag;
+import com.amazonaws.services.logs.AWSLogs;
+import com.google.common.collect.ImmutableList;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.mockito.runners.MockitoJUnitRunner;
+
+import java.util.List;
+
+import static org.junit.Assert.assertEquals;
+import static org.mockito.Matchers.any;
+import static org.mockito.Mockito.doReturn;
+import static org.mockito.Mockito.spy;
+
+@RunWith(MockitoJUnitRunner.class)
+public class DefaultEcsClientTest
+{
+    @Mock private EcsClientConfig ecsClientConfig;
+    @Mock private AmazonECSClient rawEcsClient;
+    @Mock private AWSLogs logs;
+
+    private DefaultEcsClient ecsClient;
+
+    @Before
+    public void setUp()
+    {
+        ecsClient = spy(new DefaultEcsClient(ecsClientConfig, rawEcsClient, logs));
+    }
+
+    @Test
+    public void testGetTaskDefinitionTags()
+            throws Exception
+    {
+        final List<Tag> expected = ImmutableList.of(
+                new Tag().withKey("k1").withValue("v1"),
+                new Tag().withKey("k2").withValue("v2")
+        );
+        final ListTagsForResourceResult result = new ListTagsForResourceResult()
+                .withTags(expected);
+        doReturn(result).when(rawEcsClient).listTagsForResource(any(ListTagsForResourceRequest.class));
```

Comment:
I think you could specify the ARN here, and then verify this was called with the expected parameters at the end of the test. It won't make much difference, though.

Reply:
Got it, thanks. Fixed https://github.com/treasure-data/digdag/pull/1170/commits/ac6ae52050ed5380c928043d0da8910fb0f86c0a